config/exercise_readme.go.tmpl

```diff
@@ -4,9 +4,6 @@
 {{- with .Hints }}
 {{ . }}
 {{ end }}
-{{- with .TrackInsert }}
-{{ . }}
-{{ end }}
```

Comment:
If this is now getting spliced in house, then perhaps the code for doing that could take a list of files to read and splice in.  In that case we wouldn't need to copy the `TESTS.md` file at all, and we could just reference it directly.

Reply:
Oh nevermind, I see that you elected to include altered notes on how to run the tests.

Reply:
Yeah, I think that could be a good next step. I think we should still transition incrementally, though